archetype-module-content/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      name="openmrs-archetype-module"
+                      xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <requiredProperties>
+        <requiredProperty key="groupId">
+            <defaultValue>org.openmrs.module</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="artifactId" />
+        <requiredProperty key="version">
+            <defaultValue>1.0.0-SNAPSHOT</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="package">
+            <defaultValue>${groupId}.${artifactId}</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="moduleName" />
+        <requiredProperty key="moduleDescription"/>
+        <requiredProperty key="moduleAuthor">
+            <defaultValue>mherman22</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="moduleClassnamePrefix" />
+        <requiredProperty key="openmrsContentPackageVersion"/>
+    </requiredProperties>
+
+    <fileSets>
+        <fileSet filtered="true" encoding="UTF-8">
+            <directory>.github</directory>
+            <includes>
+                <include>workflows/main.yml</include>
+            </includes>
+        </fileSet>
+        <fileSet filtered="true" encoding="UTF-8">
+            <directory/>
+            <includes>
+                <include>content.properties</include>
+                <include>.gitignore</include>
```

Comment:
Does this work? I found in the past that `.gitignore` wasn't properly copied.

Reply:
looks to be the same case here, together with the `assembly.xml` file. When i run mvn install, i get https://pastebin.ubuntu.com/p/KjrBfWYxv7/. The reference project in `target/test-classes/` doesn't contain those two files

@ibacher

Reply:
never mind this, i have not seen a fix around for the gitignore but the rest of the stuff is working well including the integration tests